pkg/store/prometheus.go

```diff
@@ -545,42 +552,54 @@ func (p *PrometheusStore) LabelNames(ctx context.Context, r *storepb.LabelNamesR
 		}
 	}
 
+	if len(lbls) > 0 {
+		for _, lbl := range extLset {
+			lbls = append(lbls, lbl.Name)
```

Comment:
```suggestion
		for _, extLbl := range extLset {
			lbls = append(lbls, extLbl.Name)
```
Nit: To make it obvious what is being appended to what